tasks/node/remotetest.js

```diff
@@ -0,0 +1,340 @@
+'use strict';
+
+require('dotenv').config({
+	silent: true
+});
+
+const path = require('path');
+const wd = require('wd');
+const denodeify = require('denodeify');
+const readFile = denodeify(require('graceful-fs').readFile);
+const writeFile = denodeify(require('graceful-fs').writeFile);
+const testResultsPath = path.join(__dirname, '../../test/results');
+const testResultsFile = path.join(testResultsPath, 'results.json');
+const mkdirp = denodeify(require('mkdirp'));
+const argv = require('minimist')(process.argv.slice(2));
+const cli = require('cli-color');
+const pollTick = 1000;
+const testBrowserTimeout = 60000;
+const environment = ['browserstack', 'saucelabs'].filter(x => x in argv)[0] || 'browserstack';
```

Comment:
Also is it needed at all or could you simplify to 

```
require('./' + ['browserstack', 'saucelabs'].filter(x => x in argv)[0] || 'browserstack');
```

And it might be slightly less obscure (and more forwards compatible) to name that CLI arg:

```
require('./' + (['browserstack', 'saucelabs'].contains(argv.provider) && argv.provider) || 'browserstack');
```